KomodoPlatform/komodo-wallet-desktop
# Enhance fiat settings

Closes: https://github.com/KomodoPlatform/komodo-wallet-desktop/issues/2307

To test:
- In settings, view "recommended fiats" row. Selected fiat should be brighter white, with other options greyed (and a slight lightening on mouse over).
- Select a different fiat from the drop down which is in the recommended row already. The highlighted option should change.
- Select a different fiat from the drop down which is not in the recommended row. It should be added to the start of the "recommended fiats" row, and be highlighted.
- Confirm selection is actually applied to app, and values are displayed in the selected fiat
